Ginger/Ginger/ApiModelsFolder/AddApiModelActionWizardPage.cs

```diff
@@ -73,6 +74,9 @@ public override void Finish()
         {
             foreach (ApplicationAPIModel aamb in AAMList)
             {
+                if (aamb.TargetApplicationKey == null || Convert.ToString(aamb.TargetApplicationKey.ItemName) != mContext.Activity.TargetApplication)
```

Comment:
this extra filtering is for the case when we Drag-Drop the API Folder from the side panel. Without this it would add all the available API's Action into the Actions List